webserver/webserver/src/main/java/io/helidon/webserver/UnsafeRequestedUriSettings.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2022 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.webserver;
+
+/**
+ * Indicates unsafe settings for a socket's request URI discovery.
+ * <p>
+ *     This exception typically results when the user has enabled requested URI discovery or selected a discovery type
+ *     but has not assigned the trusted proxy {@link io.helidon.common.configurable.AllowList}.
+ * </p>
+ */
+public class UnsafeRequestedUriSettings extends RuntimeException {
```

Comment:
An exception class should always have `Exception` suffix in the name

Reply:
changed